.github/workflows/test_installer.yml

```diff
@@ -0,0 +1,63 @@
+name: Install Local Version of Opik
+
+on:
+    workflow_dispatch:
+
+jobs:
+    test_installation:
+        runs-on: ubuntu-20.04
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v3
+              with:
+                ref: ${{ github.ref }}
+            
+            - name: Setup Python
+              uses: actions/setup-python@v5
+              with:
+                python-version: 3.12
+
+            - name: Install Opik
+              run: pip install sdks/python
+
+            - name: Install Test Dependencies
+              run: |
+                pip install -r tests_end_to_end/test_requirements.txt
+                playwright install
+
+            - name: Install Opik
+              run: |
+                cd deployment/docker-compose
+                docker compose up --detach
+
+            - name: Check Docker pods are up
+              run: |
+                cd deployment/docker-compose
+                max_retries=5
+                wait_interval=3
+                retries=0
+
+                while [[ $retries -lt $max_retries ]]
+                do
+                    containers=$(docker compose ps --status running -q)
+                    if [ -z "$containers" ]; then
+                        echo "Waiting for containers to be up... (Attempt: $((retries+1))/$max_retries)"
+                        sleep $wait_interval
+                        retries=$((retries+1))
+                    else
+                        echo "Containers running"
+                        docker compose ps
+                        break
+                    fi
+                done
+
+                if [[ $retries -eq $max_retries ]]; then
+                    echo "Containers failed to start"
+                    docker compose ps
+                    exit 1
+                fi
```

Comment:
This should all go into a script, so we can reuse it in the future. Anyway, not a blocker. We can move it in a follow-up PR.